backend/app/mailers/hubee_portail_mailer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class HubeePortailMailer < ActionMailer::Base
+  layout false
+
+  def archive_enrollment_email
+    @enrollment = Enrollment.find(params[:enrollment_id])
+
+    mail(
+      to: @enrollment.demandeurs.pluck(:email),
```

Comment:
Be careful, which demandeur do you want to send the email ? 
You can have several demandeurs and `@enrollment.demandeurs.pluck(:email)` can return an array with severals demandeurs if this one has been changed in time...